sharktank/sharktank/layers/llama_attention_block.py

```diff
@@ -5,6 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 from typing import Optional
+import math
```

Comment:
any reason to change torch.sqrt to math? historically there are problems tracing this

Reply:
Its to take sqrt of a constant (head dim value). This will always be constant at tracing time.